frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -98,6 +98,10 @@ describe('Kubernetes resource CRUD operations', () => {
 
   testObjs.forEach(
     ({ kind, namespaced = true, humanizeKind = true, testI18n = true }, resource) => {
+      // Ex: to execute just the Pod crud tests, set environment var 'cypress_k8TestResource=Pod' before running cypress
+      if (Cypress.env('k8TestResource') && kind !== Cypress.env('k8TestResource')) {
```

Comment:
Typo, it should be `k8s`

```suggestion
      // Ex: to execute just the Pod crud tests, set environment var 'cypress_k8sTestResource=Pod' before running cypress
      if (Cypress.env('k8sTestResource') && kind !== Cypress.env('k8sTestResource')) {
```

Reply:
fixed